lib/iris/cube.py

```diff
@@ -2892,6 +2906,9 @@ def _xml_element(self, doc, checksum=False, order=True, byteorder=True):
         if self.var_name:
             cube_xml_element.setAttribute('var_name', self.var_name)
         cube_xml_element.setAttribute('units', str(self.units))
+        if self.fill_value is not None:
+            cube_xml_element.setAttribute('fill_value', str(self.fill_value))
+        cube_xml_element.setAttribute('dtype', self.dtype.name)
```

Comment:
Add the cube `fill_value` and `dtype` to the CML of the cube - these attributes are akin to the metadata of the cube.